### ISSUE TITLE
Show the trivial zeros for Maass-type L-functions

### ISSUE THREAD

**Issue by davidfarmer** [LMFDB/lmfdb#2828]
The plots of L-functions with nonzero imaginary spectral parameters
make it look like RH fails.  This is because of the trivial zeros.
A big red dot on the horizontal axis at those points would help people
understand this interesting phenomenon.

Suggested by Dave Roberts.

**Comment by davidfarmer** [comment]
Refinement:

A red dot when it is Gamma_R(s + i y) and an open red circle when it is GammaR(s + 1 + i y).

In the latter case, the effect on the Z-function is less significant, as can be seen in these examples:
http://beta.lmfdb.org/L/ModularForm/GL2/Q/Maass/4f62419388aece3d9f000016/
http://beta.lmfdb.org/L/ModularForm/GL2/Q/Maass/4f622d0488aece3d9f000003/

If those links stop working because of renaming L-functions, those are the L-functions of
the Maass forms of level 9 and trivial character, with eigenvalue approximately 7.43 .

**Comment by AndrewVSutherland** [comment]
Another case noted by John Voight where the Z-plot looks wrong:

https://www.lmfdb.org/L/ModularForm/GL2/Q/Maass/5411a3c1acf7560ce8a6f2b1/

(this is for the Maass form of level 10 with spectral parameter R= 6.4845713681)

**Comment by davidfarmer** [comment]
On hold until #6235

**Comment by davidfarmer** [comment]
New cases can be done after #6236
